### ISSUE TITLE
ProgressBar not working on Linux

### ISSUE THREAD

**Issue by pmengal** [goblinfactory/konsole#81]
Here is the stack strace:

`System.PlatformNotSupportedException: Operation is not supported on this platform.
   at System.ConsolePal.MoveBufferArea(Int32 sourceLeft, Int32 sourceTop, Int32 sourceWidth, Int32 sourceHeight, Int32 targetLeft, Int32 targetTop, Char sourceChar, ConsoleColor sourceForeColor, ConsoleColor sourceBackColor)
   at System.Console.MoveBufferArea(Int32 sourceLeft, Int32 sourceTop, Int32 sourceWidth, Int32 sourceHeight, Int32 targetLeft, Int32 targetTop, Char sourceChar, ConsoleColor sourceForeColor, ConsoleColor sourceBackColor)
   at Konsole.Writer.MoveBufferArea(Int32 sourceLeft, Int32 sourceTop, Int32 sourceWidth, Int32 sourceHeight, Int32 targetLeft, Int32 targetTop, Char sourceChar, ConsoleColor sourceForeColor, ConsoleColor sourceBackColor)
   at Konsole.ConcurrentWriter.MoveBufferArea(Int32 sourceLeft, Int32 sourceTop, Int32 sourceWidth, Int32 sourceHeight, Int32 targetLeft, Int32 targetTop, Char sourceChar, ConsoleColor sourceForeColor, ConsoleColor sourceBackColor)
   at Konsole.Window.MoveBufferArea(Int32 sourceLeft, Int32 sourceTop, Int32 sourceWidth, Int32 sourceHeight, Int32 targetLeft, Int32 targetTop, Char sourceChar, ConsoleColor sourceForeColor, ConsoleColor sourceBackColor)
   at Konsole.ConcurrentWriter.MoveBufferArea(Int32 sourceLeft, Int32 sourceTop, Int32 sourceWidth, Int32 sourceHeight, Int32 targetLeft, Int32 targetTop, Char sourceChar, ConsoleColor sourceForeColor, ConsoleColor sourceBackColor)
   at Konsole.Window.ScrollDown()
   at Konsole.Window.WriteLine(String text)
   at Konsole.ConcurrentWriter.WriteLine(String text)
   at Konsole.ProgressBarSlim..ctor(Int32 max, Nullable`1 textWidth, Char character, IConsole console)
   at Konsole.ProgressBar..ctor(Int32 max, Nullable`1 textWidth, Char character, PbStyle style, IConsole console)
   at Konsole.ProgressBar..ctor(IConsole console, Int32 max, Int32 textWidth)
   at CasterStats.TvAnalytics.ConsoleProgressReport.UpdateProgressBar(ProgressReport e)
   at CasterStats.TvAnalytics.TvProgramCommand.<>c__DisplayClass0_0.<Run>b__1(Object o, ProgressReport i)
   at CasterStats.TvAnalytics.Calculate.OnReportProgress(ProgressReport e)
   at CasterStats.TvAnalytics.CalculatePrograms.<>c__DisplayClass1_0.<CalculateProgramList>b__0()
   at CasterStats.TvAnalytics.RavenDbCache`1.GetOrCreate(Object objectKey, Int32 cacheDuration, Func`1 createItem)
   at CasterStats.TvAnalytics.CalculatePrograms.CalculateProgramList(IList`1 plays, Parameters parameters)
   at CasterStats.TvAnalytics.TvProgramCommand.<Run>b__0_0(TvProgramCommandOptions parameter)
   at CasterStats.TvAnalytics.TvCommandBase`2.<>c__DisplayClass3_0.<Run>b__0(T parameter, List`1 results)
   at CasterStats.TvAnalytics.TvCommandBase`2.CalculateResult(T options, Action`2 calculateCallback)`

**Comment by goblinfactory** [comment]
This is a duplicate of #30  (Implement MoveBufferArea in Unix to enable using more cross platform support #30)
Please don't close this issue, it's useful to leave open so that other developers will read the comments and look at the issue.
Thank you
Alan

**Comment by goblinfactory** [comment]
TLDR; 

- proper cross platform support is coming ina later release. 
- If you want a cross platform progress bar please also checkout https://github.com/goblinfactory/progress-bar
